core/src/main/java/com/segment/analytics/kotlin/core/Settings.kt

```diff
@@ -118,5 +128,9 @@ internal fun Analytics.fetchSettings(
         "${ex.message}: failed to fetch settings",
         kind = LogKind.ERROR
     )
+    Telemetry.error("analytics_mobile.invoke.error",
```

Comment:
Should those Strings be CONST somewhere? Maybe we could add more context with a comment about them:

```
// This is for XYZ...
const INVOKE_ERROR = "analytics_mobile.invoke.error"
````